js/view.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var projectView = {};
+
+projectView.handleMainNav = function() {
+  $('.menu-item').on('click', function(){
+    $('.tab-content').hide();
+    $('.' + $(this).attr('data-content')).show();
```

Comment:
This works very well for your tabs, however if you click the 'GitHub' link, all the contents of your page disappear.